src/screens/ConfirmationContainer/ConfirmationScreen/index.js

```diff
@@ -27,18 +29,28 @@ type Props = {
    * @desc Function to return the Promise resolve
    * @param {number} gasPrice Number with the gasPrice selected by the user for the current transaction
    */
-  onSuccess: (gasPrice: number) => null,
+  onSuccess: (gasPrice: number, gasLimit: string) => null,
```

Comment:
Please add docs above for gasLimit parameter.

Reply:
Added